sdk/core/core-http/lib/coreHttp.ts

```diff
@@ -105,4 +105,15 @@ export { ApiKeyCredentials, ApiKeyCredentialOptions } from "./credentials/apiKey
 export { ServiceClientCredentials } from "./credentials/serviceClientCredentials";
 export { TopicCredentials } from "./credentials/topicCredentials";
 export { Authenticator } from "./credentials/credentials";
+
+export {
+  atomSerializationPolicy,
+  AtomSerializationPolicy
```

Comment:
The function that creates the policy should most definitely be exported, but what is the reason to export the policy itself?

Reply:
This was just staying consistent with way other policies were being exported, looks like that has changed now - updated.